### DISCUSSION TITLE
[PvP] A suggestion for long-term investment in PvP - From current bracket to the end of Wrath

### DISCUSSION THREAD

**Discussion by Belodina** [chromiecraft/chromiecraft#7365] [Suggestions]
_Preface: I did not actively participate during the PvP Council days or Season 1, so my suggestions may lack certain context or overlap with those made previously. However, I intend to offer a constructive and actionable proposal that I believe will incentivize both PvE and PvP players to engage more actively with one of the end-game avenues of play, PvP._

### Weekly PvP Vendor

The suggestion as an idea is simple. Add a PvP Vendor that sells limited stock reagents, consumables, raw/cut gems and rotating Transmog purchasable with Honour and/or Arena Points. (This can be expanded upon with any number of suggestions)

The idea is to simply provide an alternative incentive for players who would not normally engage in PvP activities, to try it out with something to grind for, outside of the PvP gear. Initial engagement with the system may even lead to people enjoying PvP more, and engaging further with the content and moving on to arena.

This change is directed at increasing battleground participation, as rewards can be purchased for honour, but locking more "expensive" rewards behind arena points may also give an incentive for players to play at least once daily for the AP bonus or make arena teams for the weekly arena points.

### Potential Offerings

Transmog Sets: Not normally available in 3.3.5

[Item Sets](https://wowgaming.altervista.org/aowow/?itemsets&filter=maxle=92;maxrl=70)

Reagents: Not normally offered by existing NPCs 

[Common](https://wowgaming.altervista.org/aowow/?items=7&filter=cr=87;crs=11;crv=0;qu=1;si=3;maxle=70#50-2)

[Uncommon](https://wowgaming.altervista.org/aowow/?items=7&filter=cr=87;crs=11;crv=0;qu=2;si=3;maxle=70)

[Rare](https://wowgaming.altervista.org/aowow/?items=7&filter=cr=87;crs=11;crv=0;qu=3;si=3;maxle=70)

[Epic](https://wowgaming.altervista.org/aowow/?items=7&filter=cr=87;crs=11;crv=0;qu=4;si=3;maxle=70)

Gems: [Somewhat available through PvP](https://wowgaming.altervista.org/aowow/?items=3&filter=cr=144:145;crs=1:1;crv=0:0;maxle=70)

[Gems](https://wowgaming.altervista.org/aowow/?items=3&filter=cr=88;crs=1;crv=0;qu=3;maxle=70)

Flasks, Elixirs & Potions: [Somewhat Available Through PvP](https://wowgaming.altervista.org/aowow/?items=0&filter=cr=144%3A145%3Bcrs=1%3A1%3Bcrv=0%3A0%3Bmaxle=70)

[Flasks](https://wowgaming.altervista.org/aowow/?items=0.3&filter=si=3;maxle=70)

[Elixirs](https://wowgaming.altervista.org/aowow/?items=0.2&filter=ty=2;si=3;maxle=70)

[Potions](https://wowgaming.altervista.org/aowow/?items=0.1&filter=ty=1;maxle=70)

Recipes: [Barely Available Through PvP ](https://wowgaming.altervista.org/aowow/?items=9&filter=cr=144:145;crs=1:1;crv=0:0;maxle=80)

[Gem Recipes](https://wowgaming.altervista.org/aowow/?items=9&filter=qu=3;ty=10;maxle=70)

Reputation Items:

[Fel Armaments](https://wowgaming.altervista.org/aowow/?item=29740) and [Arcane Tomes](https://wowgaming.altervista.org/aowow/?item=29739#dropped-by)

### How I envision this to work

Week 1: Vendor offers the following items and prices:

| Item | Price |
|--------|--------|
| 1x [Primal Might](https://wowgaming.altervista.org/aowow/?item=23571) | 34,700 Honour Points // 150 Arena Points |
| 2 x [Fel Lotus](https://wowgaming.altervista.org/aowow/?item=22794) | 31,000 Honour Points Each |
| 2x [Dawnstone](https://wowgaming.altervista.org/aowow/?item=23440) | 10,000 Honour Points Each |
| Example Reagent | Price |
| 5x [Flask of Relentless Assault](https://wowgaming.altervista.org/aowow/?item=22854) | 15,000 Honour Points Each |
| 5x [Flask of Pure Death](https://wowgaming.altervista.org/aowow/?item=22866) | 15,000 Honour Points Each |
| 5x [Flask of Fortification](https://wowgaming.altervista.org/aowow/?item=22851) | 15,000 Honour Points Each |
| Assorted Transmog Items Rotated Weekly | 15,000 (Uncommon) / 31,000 (Rare) / 40,000 (Epic) Honour Points | 

Week 2: The shop/offerings "rotate" to new stock

| Item | Price |
|--------|--------|
| 2x [Felsteel Bar](https://wowgaming.altervista.org/aowow/?item=23448) | 15,000 Honour Points // 75 Arena Points |
| 5x [Hardened Adamantite Bar](https://wowgaming.altervista.org/aowow/?item=23573) | 31,000 Honour Points Each |
| 2x [Living Ruby](https://wowgaming.altervista.org/aowow/?item=23436) | 10,000 Honour Points Each |
| Example Reagent | Price |
| 5x [Flask of Relentless Assault](https://wowgaming.altervista.org/aowow/?item=22854) | 15,000 Honour Points Each |
| 5x [Flask of Pure Death](https://wowgaming.altervista.org/aowow/?item=22866) | 15,000 Honour Points Each |
| 5x [Flask of Fortification](https://wowgaming.altervista.org/aowow/?item=22851) | 15,000 Honour Points Each |
| Assorted Transmog Items Rotated Weekly | 15,000 (Uncommon) / 31,000 (Rare) / 40,000 (Epic) Honour Points | 

etc.

The Vendor's offerings rotate through a set list of templates, OR draws from a random pool of possible options, resetting weekly. Feedback is welcome as some of the above prices may be too inflated. The vendor can be implemented in stages; I.e. it is added but only offers limited reagents weekly as we work on a community-accepted list of Transmogs that can be added to the vendor, etc.

For example, the prices and items available can be adjusted and determined. For example, the TBC flasks could be replaced by level 60 versions for people to use in PvP or raiding on a budget. The point of the system is to not take away from the natural avenue for obtaining reagents and consumables but to augment how a player can choose to obtain them, while also engaging players with PvP.

I believe the rewards should be compelling and have a time investment cost comparable to the time it would take to farm raw materials/grind raw gold. This should be the case to make it a competitive alternative for players as opposed to simply farming by themselves, as well as to offset the initial negative response to the idea of PvPing by some.

This PvP Vendor can be relevant at every stage of the rest of the expansion, as well as later when Northrend releases. With the exact offerings and pricing adjusted for the phase/season so as not to provide too strong of an advantage to anyone. 

### How I foresee player engagement and sentiment with the system

I believe there are many players who genuinely wish to engage with PvP, with the limiting factor being a lack of population to support an active scene, a punishing grind for gear to compete with already strong/geared players as well as a lack of rewards outside of PvP. This system is proposed with the assumption that gear prices will be adjusted to be more in line with what the community has requested so far:

https://github.com/chromiecraft/chromiecraft/discussions/7183

https://github.com/chromiecraft/chromiecraft/discussions/7276

There will be players who refuse to engage with PvP in any format and will push back at incentives being offered to PvP players which would not normally be. However, my thinking in offering a limited number of potentially rare reagents/consumables is that this is already somewhat implemented in the form of the current honour exchange for gold, as well as Marks of the Illidari dropping from raids, allowing the purchase of flasks. Most players will not have the time to farm enough honour to purchase everything from the vendor every week, which acts as a further reward to the most dedicated PvP players being able to "cap" their vendor. 

The time which might've been spent farming ores/herbs/gold could be augmented by the fun of playing PvP, which might be a relief week-to-week for people who would like a break from farming.

### Feasibility and Implementation

Now I am a musician by trade, so my understanding of code, SQL and its implementation in an ongoing WoW server is limited. However, I am drawing this idea from systems I believe are already implemented in the game:

- Vendors with limited items and "restocking times". I am not sure if it is possible to set vendors to restock at certain times, instead of after a set time after purchase. Furthermore, I am not sure if it is possible to have the limited stock show per individual character that interacts with the vendor. 

- The Roboto (?) Honour for Glyph Vendor indicates that this is already something that can be implemented in some capacity, outside of the rotating offerings/limited stock.

Furthermore, there would have to be community feedback prior to any official work on an addition like this (assuming there is not something already being worked on that clashes with this or fulfils a similar role, that is unannounced). Once initial community sentiment is gauged, the Vendor could be implemented in a limited capacity (ie only offering limited stock reagents and/or gems, as has been suggested before https://github.com/chromiecraft/chromiecraft/discussions/7109) and after live player feedback is generated, development of the Vendor can be furthered, or the Vendor can be removed. 

While I do not engage much in Transmog, I believe there are many individuals who would have valuable feedback on what Transmog items and sets could potentially be included in the vendor's catalogue. I would not be opposed to doing the legwork for the staff in creating a form and gathering suggestions, as well as tabulating items with Aowow links and prices for addition to the vendor, if it gets off the ground.

### Why do I think this will work if Honour for Gold wasn't enough of an incentive?

Firstly, I believe the population slump is past the worst, and we will see a rise in population towards the end of TBC, with an influx of new players as well as returning players. By capitalising on this and incentivising one of the end-game avenues of play, we set the stage for improving player retention, as well as an additional point of interest for drawing new players to the server. It is generally known that PvP on ChromieCraft is not really competitive and is not regarded highly. By acknowledging the community, and adding non-Blizzlike rewards/incentives, I believe it is possible to change this outlook and foster positive sentiment instead of allowing the PvP community to wallow in :dedge:

Furthermore, with the AH in the state that it is, this also acts as an avenue to further stimulate the server economy outside of giving raw gold to individuals. Additionally, the "minigame" of getting limited items that may not be available the next week is also a fun aspect which Honour for Gold does not provide.

I believe the Transmog will be the determining factor in the success of such a system, if this gets off the ground.

### Conclusion

While I fully support the server's goals of recreating the version of the game, as it was in 3.3.5, this change addresses some of the issues that arise with Blizzard's systems on a lower-population server. I believe that the addition of this change is evergreen and will provide a positive benefit to the server and further its goals for the rest of TBC, as well as during Wrath of the Lich King. However, it is always possible to remove aspects of the vendor in Wrath of the Lich King to fall more in line with traditional progression and rewards avenues (although I think the Transmog aspect should stay as a bonus incentive for PvP).

I encourage PvP players who genuinely wish for the improvement of the server population, as well as the longevity of PvP as an avenue of play to respond with feedback as well as improvements to the system. This post does not address other issues with PvP, such as balance at level 70, and only wishes to address the possibility of incentivising PvP for PvE players. I do not wish for this to serve as a space for people to air personal grievances and I hope that we can genuinely improve the current state of PvP.

_I am aware that there are several plans for PvP currently in action by the staff, the above is a personal idea I had based on previous PvP suggestions, (hopefully) constructively formatted and aimed at generating positivity in the PvP headspace. Please feel free to let us know if the above overlaps with something already in action, or is not feasible._

**Comment by TotallyNotMyAccount** [comment]
The cost and what to sell in general can be discussed further, but I approve of the idea in general.

**Comment by SignorSimone** [comment]
I agree.

**Comment by walkingminterly** [comment]
updooot

**Comment by UndoUreche** [comment]
If a new vendor is added, please make them blend in with the arena npcs. Up up this goes

**Reply by Belodina** [reply]
Gonna see about installing a local AC and adding the Vendor myself this weekend so the Devs have less work to do. Would be perfect to have a goblin NPC as the vendor!
